app/components/action-feed-entry/style.scss

```diff
@@ -1,10 +1,6 @@
 & {
   clear: both;
 
-  .entry-body {
-    padding: 20px 15px;
```

Comment:
Why no padding?
![screen shot 2017-11-06 at 4 46 51 pm](https://user-images.githubusercontent.com/6776190/32465674-45776420-c312-11e7-81c7-752735b48d30.png)

Reply:
Aw...I forgot about the hover box. The top of the list looks out of line with the provider box with top padding.